src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -254,6 +344,30 @@ export class QuickBar extends LitElement {
             --mdc-dialog-max-height: calc(100% - 72px);
           }
         }
+
+        ha-icon {
+          color: var(--secondary-text-color);
+        }
```

Comment:
@bramkragten Why is this `ha-icon` styling no longer in `dev`?

Current dev:
https://github.com/home-assistant/frontend/blob/07fc9b98cca2d9e9dd919ebaed080560ce4db8f0/src/dialogs/quick-bar/ha-quick-bar.ts#L338-L358

I looked through git blame and the commit history of ha-quick-bar and I don't see anything that removed it O_o

Reply:
It probably doesn't matter, icons look okay in dark mode.

But it does seem spooky that this code could vanish without a github trace.